cheeseshop/dbapi.py

```diff
@@ -311,10 +311,79 @@ class CsGoSteamId(object):
         await conn.execute('''
             CREATE TABLE cs_go_steam_ids(
                 id serial PRIMARY KEY,
-                steam_id text UNIQUE NOT NULL
+                steam_id text UNIQUE NOT NULL,
+                print_name text,
+                team integer REFERENCES cs_go_team_ids (id)
             )
         ''')
 
+    @staticmethod
+    async def get_all(conn):
+        players = []
+        async for record in conn.cursor('''
+            SELECT * from cs_go_steam_ids
+        '''):
+            players.append({
+                'id': record['steam_id'],
+                'name': record['print_name'],
+                'team': record['team_name}']})
+        return players
+
+
+class CsGoTeamNames(object):
+    @staticmethod
+    async def create_schema(conn):
+        await conn.execute('''
+            CREATE TABLE cs_go_team_names(
+                id serial PRIMARY KEY,
+                team integer REFERENCES cs_go_team_ids (id),
+                team_name text
+            )
+        ''')
+
+    @staticmethod
+    async def get_names_by_id(conn, team_id):
+        names = []
+        async for record in conn.cursor('''
+            SELECT * FROM cs_go_team_names
+            WHERE id = $1
+        ''', team_id):
+            names.append(record['team_name'])
+        return names
+
+    @staticmethod
+    async def get_ids_by_name(conn, team_name):
+        ids = []
+        async for record in conn.cursor('''
+            SELECT * FROM cs_go_team_names
+            WHERE team_name LIKE %$1$%
+        ''', team_name):
+            ids.append(record['team'])
+        return ids
+
+    def __init__(self, id_, name):
+        self.id = id
+        self.name = name
+
+
+class CsGoTeam(object):
+    @staticmethod
+    async def create_schema(conn):
+        await conn.execute('''
+            CREATE TABLE cs_go_team_ids(
```

Comment:
Lets just call this table cs_go_teams (id isnt an external property here like steamid, its a key were using internally).

Reply:
Ok.